packages/schema-dot-org-types/tools/transform-schema.js

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env node
+// @flow
+
+import JSONStream from "JSONStream";
+import eventStream from 'event-stream';
+import request from 'request';
+import template from 'url-template';
+import fs from 'fs';
+import TypeCollector from './type-collector';
+import TypeGenerator from './class-generator';
+import program from 'commander';
+import path from 'path';
+
+const SCHEMA_ORG_URL = template.parse('http://schema.org/{type}.jsonld');
+
+function fetchType(url: string, processor : Function) : Promise<*> {
+    return processType(
+        request({ url }),
+        processor);
+}
+
+function readType(path : string, processor : Function) : Promise<*> {
+    return processType(
+        fs.createReadStream(path),
+        processor);
+}
+
+function processType(stream: any, processor : Function) : Promise<*> {
+    return new Promise(function (resolve, reject) {
+        const events = eventStream.map(function (elem, callback) {
+            const result = processor(elem);
+            callback(null, result);
+        });
+        stream.pipe(JSONStream.parse('@graph.*'))
+            .pipe(events);
+        events.on('error', function (error: Error) {
+            reject(error);
+        });
+        events.on('end', function () {
+            resolve();
+        })
+    });
+}
+
+let commandName = process.argv[0];
+if (commandName.match(/node$/)) {
+    commandName = path.basename(process.argv[1]);
+}
+const USAGE = `${commandName} [options] <schema-type>`;
```

Comment:
You need to have a `bin` field in package.json for this to actually be addressable as an executable when installed